guides/common/modules/ref_overview-of-the-host-columns.adoc

```diff
@@ -0,0 +1,46 @@
+[id="overview-of-the-host-columns_{context}"]
+= Overview of the Host Columns
+
+Below is the complete overview of columns that can be displayed in the host table divided into content categories.
+
+General::
+* Power - Whether the host is turned on or off, if available
+* Name - name of the host
+* Operating system - operating system of the host
+* Owner - user or group owning the host
+* Host group - host group of the host
+* Boot time - last boot time of the host
+* Last report - time of the last host report
+* Comment - comment given to host
+
+ifdef::katello,satellite,orcharhino[]
+Content::
+* Name - name of the host
+* Operating system - operating system of the host
+* Subscription status - does the host have a valid subscription attached.
+* Installable updates - numbers of installable updates divided into four categories: security, bugfix, enhancement, total
+* Lifecycle Environment - lifecycle environment of the host
+* Content view - content view of the host
+* Registered - time when the host was registered to satellite.
```

Comment:
Some descriptions end with a dot, some don't. Not sure which is correct, but we shouldn't do both

Reply:
Thanks for noticing! I think I will remove the dot from everywhere.